src/components/augmentedReality/AugmentedReality.js

```diff
@@ -1,29 +1,28 @@
 import PropTypes from 'prop-types';
 import React, { useEffect, useState } from 'react';
 import { isARSupportedOnDevice } from '@viro-community/react-viro';
+import { useQuery } from 'react-apollo';
 
 import { consts, device, texts } from '../../config';
 import { checkDownloadedData } from '../../helpers';
-import { useStaticContent } from '../../hooks';
 import { Button } from '../Button';
 import { LoadingSpinner } from '../LoadingSpinner';
 import { Title, TitleContainer, TitleShadow } from '../Title';
 import { Wrapper } from '../Wrapper';
+import { GET_TOUR_STOPS } from '../../queries/tours';
 
 import { ARModal } from './ARModal';
 import { ARObjectList } from './ARObjectList';
 import { WhatIsARButton } from './WhatIsARButton';
+import { getQuery, QUERY_TYPES } from '../../queries';
 
-export const AugmentedReality = ({ navigation, onSettingsScreen, tourID }) => {
+export const AugmentedReality = ({ id, navigation, onSettingsScreen }) => {
   const {
-    data: staticData,
-    error,
-    loading,
-    refetch
-  } = useStaticContent({
-    name: `arDownloadableDataList-${tourID}`,
-    type: 'json'
-  });
+    data: {
+      tour: { tourStops }
```

Comment:
is this safe? i think it could be a problem destructing for `tourStops` if there is no `tour` in `data`. how can you be sure, that this does not happen?

Reply:
Hmm. I'll have to think of something for that.

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/608/commits/afa7f49f3c7659b48f0bc217d6e549966e443c24